src/storage/gc_worker.rs

```diff
@@ -212,25 +264,130 @@ impl<E: Engine> GCRunner<E> {
         );
         Ok(())
     }
-}
 
-impl<E: Engine> Runnable<GCTask> for GCRunner<E> {
-    fn run(&mut self, mut task: GCTask) {
-        GC_GCTASK_COUNTER.inc();
+    fn unsafe_destroy_range(&self, _: &Context, start_key: &Key, end_key: &Key) -> Result<()> {
+        info!(
+            "unsafe destroy range start_key: {}, end_key: {} started.",
+            start_key, end_key
```

Comment:
Please escape the key.

Reply:
It's already escaped in `impl Display for Key`